src/CoreWCF.Primitives/tests/Helpers/TestHelper.cs

```diff
@@ -1,16 +1,11 @@
-﻿using CoreWCF.Channels;
-using CoreWCF.Configuration;
-using Helpers;
-using Microsoft.AspNetCore.Hosting.Server;
-using Microsoft.AspNetCore.Hosting.Server.Features;
-using Microsoft.Extensions.DependencyInjection;
+﻿using CoreWCF;
+using CoreWCF.Channels;
 using System;
-using System.Collections.Generic;
+using System.IO;
 using System.Text;
-using System.Threading;
 using System.Xml;
 
-namespace CoreWCF.Primitives.Tests.Helpers
+namespace Helpers
```

Comment:
What is the rationale of making Helpers a "top level" namespace?

Reply:
None of these api's are being consumed by other libraries and the really long namespace just seems a little obnoxious to me. The purpose of the namespace is to prevent things like System.ServiceModel.ChannelFactory from being confused with CoreWCF.ChannelFactory when end user code is referencing both assemblies but that's not the case here.

Reply:
If you feel strongly about it, let me know and I'll put this back and follow up with an other PR to update all the other namespaces to have the project name in them too.

Reply:
@tibor19, if you want this changed. you can add a comment post merge and I'll open a PR to fix all the top level namespaces. For now I'm going to merge this with the other things fixed.